lib/readline.js

```diff
@@ -48,6 +48,7 @@ function Interface(input, output, completer, terminal) {
 
   EventEmitter.call(this);
   var historySize;
+  var deDupeHistory;
```

Comment:
Done. I originally did not initialize the variable to match the style of the `historySize` variable declaration that precedes it, but this change is immaterial.